org.groestlcoin.groestlcoin-qt.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "org.groestlcoin.groestlcoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "19.08",
+  "command": "groestlcoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    "--socket=wayland",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "groestlcoin-core",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/groestlcoind",
+        "install -m 0744 -D -t /app/bin/internal bin/groestlcoin-qt",
+        "install -m 0744 -D -t /app/bin/internal bin/groestlcoin-cli"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
```

Comment:
Can you add flathub.json file with `only-arches` set to available architectures? Just in case we add a new arch one day.

https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
Done